### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by bpalmintier**
Addresses #101

### CODE REVIEW

**Comment by bpalmintier** [comment]
@phlptp: You might want to look at how Griddyn is referenced, particularly the note about where/how HELICS support is provided. Might require some updates depending on how you plan to publish.

File: README.md [hunk 1]
```diff
@@ -18,3 +22,58 @@ Repository for the source code for the Hierarchical Engine for Large-scale Infra
...
+
+We are always looking for help adding support for more tools, so please contact us if you have any additions.
+
+#### Power systems Tools
+
+* [GridLAB-D](https://www.gridlabd.org/), distribution power-flow, DER models, basic house thermal and end-use load models, and more. HELICS support currently provided in the [`feature/1024` branch](https://github.com/gridlab-d/gridlab-d/tree/feature/1024) which you have to build yourself as described here
+* [Griddyn](https://github.com/LLNL/GridDyn), a transmission power flow and dynamics simulator. HELICS support currently provided through an experimental branch.
+* [PSST](https://github.com/kdheepak/psst), a python-based unit-commitment and dispatch market simulator. HELICS examples are included in the  [HELICS-Tutorial](https://github.com/GMLC-TDC/HELICS-Tutorial)
+* InterPSS, a Java-based power systems simulator.
```
Comment (phlptp) [inline README.md]: add link to interpss, how far along is the integration with HELICS?

File: README.md [hunk 2]
```diff
@@ -18,3 +22,58 @@ Repository for the source code for the Hierarchical Engine for Large-scale Infra
...
+#### Communication Tools
+
+* [ns-3](https://www.nsnam.org/), a discrete-event communication network simulator. Supported via the [HELICS fork of ns-3](https://github.com/GMLC-TDC/ns-3-dev-git)
+* HELICS also includes built-in support for simple communications manipulations such as delays, lossy channels, etc. through its built-in filters.
+
+#### HELICS helper Apps
+
+* [HELICS runner](https://github.com/GMLC-TDC/helics-runner) provides a simple way to automate configuring, starting, and stopping HELICS co-simulations.
+* [Player](https://gmlc-tdc.github.io/HELICS-src/apps/Player.html), which acts as a simple send-only federate that simply publishes a stream of timed HELICS messages from a user-defined file. HELICS Player is included in the HELICS distribution.
+* [Recorder](https://gmlc-tdc.github.io/HELICS-src/apps/Recorder.html), which acts a s simple received-only federate that prints out or saves messages from one or more subscribed streams. HELICS Recorder is included in the HELICS distribution.
```
Comment (phlptp) [inline README.md]: fix spelling "a s simple"

File: README.md (hunk 1 above)
Comment (bpalmintier) [reply README.md]: Done, also MATPower added. all in 58ceae818a75eaffa6b2d5a36ac5643712e4fc3c

File: README.md (hunk 2 above)
Comment (bpalmintier) [reply README.md]: Done: b441a7e4d880cf0dcac124d10cad6e07b5a3ee64

**Comment by bpalmintier** [comment]
OK think we are set. Most recently added specific GridDyn branch link, per @phlptp and put dates on info that is likely volatile with time.

**Pull request merged**